certbot/storage.py

```diff
@@ -204,7 +207,7 @@ class RenewableCert(object):  # pylint: disable=too-many-instance-attributes
         renewal configuration file and/or systemwide defaults.
 
     """
-    def __init__(self, config_filename, cli_config):
+    def __init__(self, config_filename, cli_config, update_symlinks=False):
```

Comment:
As above, how come `update_symlinks` belongs in the constructor? Is there something particular about instantiating `RenewableCert` that should make this action sometimes get invoked here as opposed to be (perhaps immediately) calling a method on the resulting instantiated object? (I'm just wary of complicating the API.)